packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -409,8 +409,8 @@ export class TreeViewWidget extends TreeViewWelcomeWidget {
     @inject(ContextKeyService)
     protected readonly contextKeyService: ContextKeyService;
 
-    @inject(TooltipService)
-    protected readonly tooltipService: TooltipService;
+    @inject(HoverService)
```

Comment:
I believe that with the use of the HoverService here, we are no longer making use of the markdownIt field of this widget, and it could likely be at least deprecated, or more probably removed?

Reply:
I believe we can safely remove it, good catch 👍